src/Fakes/v1/YousignFaker.php

```diff
@@ -71,8 +71,10 @@ class YousignFaker
 
     /**
      * Constructor for `YousignFaker` class.
+     *
+     * @param array<string,mixed> $customFakers
```

Comment:
```suggestion
     * @param array<array-key,Response> $customFakes
```

Dovrai importare il modello giusto che è il tipo che ti torna `Http::response()` credo.